src/rules/converters/quotemark.ts

```diff
@@ -0,0 +1,32 @@
+import { RuleConverter } from "../converter";
+
+export const convertQuotemark: RuleConverter = tslintRule => {
+    const notices: string[] = [];
+    const ruleArguments: any[] = [];
+
+    ["jsx-single", "jsx-double", "avoid-template"].forEach(rule => {
```

Comment:
> `rule`

This confused me for a few seconds: really we're looping over the _options_ for the rule, not rules themselves. Could you please rename these `rule` variables to `option`?

Reply:
Done